be/src/exprs/vectorized/string_functions.cpp

```diff
@@ -2147,6 +2147,18 @@ ColumnPtr StringFunctions::get_char(FunctionContext* context, const Columns& col
     return VectorizedStringStrictUnaryFunction<get_charImpl>::evaluate<TYPE_INT, TYPE_CHAR>(columns[0]);
 }
 
+// strcmp
+DEFINE_BINARY_FUNCTION_WITH_IMPL(strcmpImpl, lhs, rhs) {
+    if (lhs == rhs) {
+        return 0;
+    }
+    return lhs > rhs ? 1 : -1;
```

Comment:
if lhs is not equal with rhs, this function compares these two strings twice, which is not efficient.
There is already compare function in Slice, which you can use now.

Reply:
Thanks @imay for your comments. I updated it.